docs/source/app-auth-update.rst

```diff
@@ -0,0 +1,42 @@
+Updating an App's credentials
+=============================
+
+Overview
+--------
+
+You may need to change an App's credentials due to internal rotation policies or otherwise. The StreamAlert CLI allows you to easily update App credentials.
+to aid in this process, the CLI also give you the ability to list currently configured Apps.
+
+Listing Apps
+------------
+
+To list currently configured StreamAlert Apps (grouped by cluster), use the CLI command:
+
+.. code-block:: bash
+
+  $ python manage.py app list
+
+This list will be empty for each cluster if no apps are configured, or will appear similar to the following::
+
+  Cluster: prod
+
+    Name: duo_prod_collector
+      log_level:                     info
+      interval:                      rate(2 hours)
+      timeout:                       80
+      memory:                        128
+      current_version:               $LATEST
+      type:                          duo_auth
+
+
```

Comment:
`Note: The output will be empty if you haven't configured and deployed any StreamAlert Apps.`